www/templates/intro/intro.html

```diff
@@ -1,11 +1,6 @@
 <ion-view hide-nav-bar="true" class="intro-view">
     <ion-slide-box delegate-handle="intro-box" on-slide-changed="slideChanged(index)" show-pager="false">
-      <ion-slide>
-        <ng-include src="'templates/intro/summary.html'"></ng-include>
-      </ion-slide>
-      <ion-slide>
-        <ng-include src="'templates/intro/consent.html'"></ng-include>
```

Comment:
wait, did you intend to remove the summary and consent?
Why make changes to them then?